data/Sweedman_2006/metadata.yml

```diff
@@ -0,0 +1,602 @@
+source:
+  primary:
+    key: Sweedman_2006
+    bibtype: Book
+    year: 2006
+    author: Luke Sweedman and David Merritt
+    title: 'Australian Seeds: A Guide to Their Collection, Identification, and Biology'
+    publisher: CSIRO Publishing
+    isbn: '9780643094079'
+contributors:
+  data_collectors:
+  - last_name: Sweedman
+    given_name: Luke
+    ORCID: .na
+    affiliation: Western Australian Seed Centre, Botanic Garden and Parks Authority,
+      Western Australia
+  - last_name: Merritt
+    given_name: David
+    ORCID: 0000-0002-3250-6861
+    affiliation: Curtin University, Australia
+  - last_name: Fryer
+    given_name: Roger
+    ORCID: .na
+    affiliation: Western Australian Seed Centre, Botanic Garden and Parks Authority,
+      Western Australia
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: no
+  custom_R_code:     '
+    data %>% 
+      mutate(across(c("quickest time to germinate", "longest time to germinate"), ~na_if(.x, 0)))
```

Comment:
I don't think there are any NAs in these columns to begin with

Reply:
no, but there were 0's which can't be real values, so I was converting them to NA's so they don't pop up in the excluded data table

Reply:
My bad! I thought you were replacing NAs with 0s but you were doing the opposite